src/services/socket.tsx

```diff
@@ -0,0 +1,169 @@
+'use client'
+
+import { io } from 'socket.io-client'
+import { useEffect, useState } from 'react'
+
+export type ListenerRes = {
```

Comment:
q: Why do you export this? I can tell you have this bad habit of exporting all stuff for no reason. Be mindful of that. Only export what's really necessary :) It's easier for in big codebases for maintainence, for example: Refactors are easier without worrying about other files. Better auto-completed with only the necessary things.

Reply:
I'll change that. I exported it because I had assumed I would need to use it elsewhere. I didn't realize that it would end up only being used in this file. 

I did a search in the repo, that is the only use of export that is unneeded. Everything else that is exported is used in another file.

Reply:
DONE